css-squid/src/main/java/org/sonar/css/CssAstScanner.java

```diff
@@ -52,11 +53,18 @@ private CssAstScanner() {
   /**
    * Helper method for testing checks without having to deploy them on a Sonar instance.
    */
-  public static SourceFile scanSingleFile(File file, SquidAstVisitor<LexerlessGrammar>... visitors) {
+  public static SourceFile scanSingleFile(File file, CssConfiguration configuration, SquidAstVisitor<LexerlessGrammar>... visitors) {
```

Comment:
this method is for testing only, why dont we create another signature instead and we dont need to change all the tests.